### ISSUE TITLE
Cloning hangs when access is given trough iyo groups

### ISSUE THREAD

**Issue by grimpy** [gigforks/gogs#28]
When a user only has access trough a IYO group cloning hangs indefintly

```
root@js9:/opt/code/docs.greenitglobe.com/openvcloud# git clone  ssh://git@docs.greenitglobe.com:10022/openvcloud/openvcloud.git
Cloning into 'openvcloud'...
```

**Comment by LeeSmet** [comment]
To fix this we will need to hook into the cloning handler and add some functionality to see if the user is part of an itsyouonline organization

**Comment by Dinaamagdy** [comment]
solved on gitea 1.0.0
